ibm_cloud_sdk_core/base_service.py

```diff
@@ -136,6 +159,14 @@ def configure_service(self, service_name: str) -> None:
             )
         if config.get('ENABLE_GZIP') is not None:
             self.set_enable_gzip_compression(config.get('ENABLE_GZIP') == 'True')
+        if config.get('MAX_RETRIES'):
```

Comment:
I think we should support the same set of service-level config properties that we support in the Go core:
ENABLE_RETRIES (boolean)
MAX_RETRIES
RETRY_INTERVAL
where ENABLE_RETRIES must be set to true for retries to be enabled, in which case we look at the values for the other two properties and use those for configuration if present (otherwise we use default values).
So to be clear, if ENABLE_RETRIES is set to true and is the only property set, then retries are enabled with defaults for max retries and the interval.